private-pools-example/cloudbuild.yaml

```diff
@@ -0,0 +1,7 @@
+steps:
+- name: 'gcr.io/cloud-builders/curl'
+  args: ['<add-ip-address>:80']
+
+# run build on private pool
+options:
```

Comment:
use the GA verison of the config: https://cloud.google.com/build/docs/private-pools/run-builds-in-private-pool#running_builds

Reply:
Updated in https://github.com/GoogleCloudPlatform/cloud-build-samples/pull/13/commits/97d3ee671c16fbf95a5ddbce21d0b41a466cf2ab